app/modules/listings/components/listing_element_show.svelte

```diff
@@ -105,24 +170,39 @@
     margin: 0 1em;
     flex: 2;
   }
-  button{
-    min-width: 10rem;
-    margin: 0 0.5em;
+  .buttons-wrapper{
+    @include display-flex(row, center, space-between);
+    margin-block-start: 1.5em;
   }
-  .remove-button{
-    margin: 1em 0;
+  :global(.dropdown-content){
+    @include radius;
+  }
+  .element-button{
+    min-width: 10rem;
     padding: 0.4em 0.6em;
-    @include shy(0.9);
-    @include display-flex(row, center, center);
+    @include shy(0.6);
     :global(.fa){
       font-size: 1.1em;
     }
     &:hover, &:focus{
       border-radius: $global-radius;
+      background-color: $light-grey;
+    }
+  }
+  .remove-button{
+    &:hover, &:focus{
       background-color: $danger-color;
-      color: white;
     }
   }
+  [role="menu"] button{
+    @include shy-border;
+    @include tiny-button-padding;
+    @include bg-hover(white, 5%);
+    padding: 1em;
```

Comment:
[non-blocking]
That produces big buttons; what about giving them the same aspect as when displayed within the dot dot dot menu (minus the checkbox)?

Reply:
done in 75ae9a946 significantly smaller than last version, but i think the dotdotdot menu items are too small 75ae9a946